### DISCUSSION TITLE
Private Key

### DISCUSSION THREAD

**Discussion by volvoxx8** [IBAX-io/go-ibax#646] [Soru Cevap - Türkçe]
Merhaba zamanında private ve public keyleri kaydetmeyenler, yeni cüzdan oluşturduklarında wallet adress ID yenileniyor ama site üzerinde eski wallet adres kalıyor dolayısıyla kazandığımız puanları altarabileceğmiz yeni cüzdanı tekrar alsak bile siteye tanımlıyamıyoruz. Çözüm öneriniz var mı
![16430627442671417791373385895630](https://user-images.githubusercontent.com/98360746/150874509-89af310a-a1e5-47cb-8d7f-8a80f5ddcc3d.jpg)

![16430626114371433596466474184137](https://user-images.githubusercontent.com/98360746/150874226-c9f63f1d-3c16-4aec-8bf8-ae3a4372b6c2.jpg)

**Comment by Yusufknk** [comment]
Eski cüzdanın kelimelerini ve şifrelerini yeni cüzdan kısmından girerek eski cüzdanınıza erişim imkanı vardır. Bu yöntem birçok kişide olumlu sonuç verdi.
